src/main/java/org/jenkinsci/plugins/zap/ZAPBuilder.java

```diff
@@ -338,7 +342,7 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
             listener.error(ExceptionUtils.getStackTrace(e));
             return false;
         }
-        return res;
+        return true;
```

Comment:
I think it should be the best way. I don't know, if we could get the boolean value of Result res.